src/access_nri_intake/cli.py

```diff
@@ -218,3 +219,51 @@ def _get_project(path):
     if update:
         with open(os.path.join(_here, "data", "catalog.yaml"), "w") as fobj:
             yaml.dump(yaml_dict, fobj)
+
+
+def metadata_validate():
+    """
+    Check provided metadata.yaml file(s) against the experiment schema
+    """
+
+    parser = argparse.ArgumentParser(description="Validate a metadata.yaml file")
+    parser.add_argument(
+        "file",
+        type=str,
+        help="The path to the metadata.yaml file (can include wildcards for multiple metadata.yaml)",
```

Comment:
Wildcards will need to be quoted. Or you could offload the `glob` to the shell and accept arrays of files as inputs.

Reply:
True - in fact I've stung myself with this a few times. I'll add a comment.

Reply:
Actually, decided to offload the `glob` to the shell since that makes usage easier/clearer